### PR TITLE
cloud: use node-level resource metrics for monitoring

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qiancai — APPROVED** [review]
LGTM

**Comment by qiancai** [comment]
@qiffang, PTAL, thanks.

File: tidb-cloud/built-in-monitoring.md [hunk 1]
```diff
@@ -93,19 +93,19 @@ The following sections illustrate the metrics on the Monitoring page for Dedicat
...
-| TiKV Memory Usage | instances | The memory usage statistics of each TiKV instance. |
-| TiKV IO MBps | instances-write, instances-read | The total bytes of read and write in each TiKV instance. |
-| TiKV Storage Usage | instances | The storage size per TiKV instance. |
-| TiFlash Uptime | instances | The runtime of each TiFlash instance since last restart. |
-| TiFlash CPU Usage | instances | The statistics of CPU usage of each TiFlash instance. |
-| TiFlash Memory  | instances | The memory usage statistics of each TiFlash instance. |
-| TiFlash IO MBps | instances-write, instances-read | The total bytes of read and write in each TiFlash instance. |
-| TiFlash Storage Usage | instances | The storage size per TiFlash instance. |
+| TiDB Uptime | node | The runtime of each TiDB node since last restart. |
+| TiDB CPU Usage | node | The statistics of CPU usage of each TiDB node. |
```
Comment (hfxsd) [inline tidb-cloud/built-in-monitoring.md:97]:
```suggestion
| TiDB CPU Usage | node | The CPU usage statistics of each TiDB node. |
```

File: tidb-cloud/built-in-monitoring.md [hunk 2]
```diff
@@ -93,19 +93,19 @@ The following sections illustrate the metrics on the Monitoring page for Dedicat
...
+| TiDB Memory Usage | node | The memory usage statistics of each TiDB node. |
+| TiKV Uptime | node | The runtime of each TiKV node since last restart. |
+| TiKV CPU Usage | node | The statistics of CPU usage of each TiKV node. |
```
Comment (hfxsd) [inline tidb-cloud/built-in-monitoring.md:100]:
```suggestion
| TiKV CPU Usage | node | The CPU usage statistics of each TiKV node. |
```

File: tidb-cloud/built-in-monitoring.md [hunk 3]
```diff
@@ -93,19 +93,19 @@ The following sections illustrate the metrics on the Monitoring page for Dedicat
...
+| TiKV Memory Usage | node | The memory usage statistics of each TiKV node. |
+| TiKV IO MBps | node-write, node-read | The total bytes of read and write in each TiKV node. |
+| TiKV Storage Usage | node | The storage size per TiKV node. |
```
Comment (hfxsd) [inline tidb-cloud/built-in-monitoring.md:103]:
```suggestion
| TiKV Storage Usage | node | The storage usage statistics of each TiKV node. |
```

File: tidb-cloud/built-in-monitoring.md [hunk 4]
```diff
@@ -93,19 +93,19 @@ The following sections illustrate the metrics on the Monitoring page for Dedicat
...
+| TiFlash Uptime | node | The runtime of each TiFlash node since last restart. |
+| TiFlash CPU Usage | node | The statistics of CPU usage of each TiFlash node. |
```
Comment (hfxsd) [inline tidb-cloud/built-in-monitoring.md:105]:
```suggestion
| TiFlash CPU Usage | node | The CPU usage statistics of each TiFlash node. |
```

File: tidb-cloud/built-in-monitoring.md [hunk 5]
```diff
@@ -93,19 +93,19 @@ The following sections illustrate the metrics on the Monitoring page for Dedicat
...
+| TiFlash Memory  | node | The memory usage statistics of each TiFlash node. |
+| TiFlash IO MBps | node-write, node-read | The total bytes of read and write in each TiFlash node. |
+| TiFlash Storage Usage | node | The storage size per TiFlash node. |
```
Comment (hfxsd) [inline tidb-cloud/built-in-monitoring.md:108]:
```suggestion
| TiFlash Storage Usage | node | The storage usage statistics of each TiFlash node. |
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

**Review by mashenjun — APPROVED** [review]

**Pull request merged**